Bottom: a4dc683f08dd3dd8c3c41946ef161aafe7bb26f6
Top:    b966b261ccd1a82255f83c5c47acfcde1f604e7f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-25 16:32:56 +0000

Refresh of only-assign-stencil-planes-for

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index aeaa387..3f02ff7 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -214,7 +214,6 @@ ghid_set_layer (const char *name, int group, int empty)
   glEnable (GL_STENCIL_TEST);                   // Enable Stencil test
   glStencilOp (GL_KEEP, GL_KEEP, GL_REPLACE);   // Stencil pass => replace stencil value (with 1)
   hidgl_return_stencil_bit (stencil_bit);       // Relinquish any bitplane we previously used
-//  if (SL_TYPE (idx) != SL_FINISHED) {
   if (group_visible) {
     stencil_bit = hidgl_assign_clear_stencil_bit();       // Get a new (clean) bitplane to stencil with
     glStencilMask (stencil_bit);                          // Only write to our subcompositing stencil bitplane
